blog/2025-09-24-gravitino-1-0-0-release-notes.mdx

```diff
@@ -0,0 +1,128 @@
+
+Please refer to the policy system [documentation](https://gravitino.apache.org/docs/1.0.0/manage-policies-in-gravitino) to know how to use it. If you want to know more about the implementation details of the policy system, please see [#7139](https://github.com/apache/gravitino/issues/7139).
+
+### Job system
+
+The job system is another system for you to submit and run jobs through Gravitino. Users can 
+register a job template, then trigger a job based on the specific job template. Gravitino will help to submit the job to the dedicated job executor, like Apache Airflow. Gravitino can manage the job lifecycle and save the job status in it. With the job system, users can run a self-defined job to accomplish a metadata-driven action system.
+
+The job system is still in active development. In 1.0.0, we have an initial version to support running the jobs as a local process. If you want to know more about the design details, you can follow issue [#7154](https://github.com/apache/gravitino/issues/7154). Also, a user-facing documentation can be found [here](https://gravitino.apache.org/docs/1.0.0/manage-jobs-in-gravitino).
+
+The whole metadata-driven action system is still in an alpha phase for version 1.0.0. The community will continue to evolve the code and take the Iceberg table maintenance as a reference implementation in the next version. Please stay tuned.
+
+## Agent-ready through the MCP server
+
+MCP is a powerful protocol to bridge the gap between human languages and machine interfaces. With MCP, users can use natural language to communicate with the LLM, and the LLM can understand the context and invoke the right tools.
+
+With Gravitino 1.0.0, the community officially delivered the MCP server for Gravitino. Users can launch it as a remote or local MCP server and connect to the different MCP applications, like Cursor and Claude Desktop. Also, we exposed all the metadata-related interfaces as tools, which can be called by MCP clients.
+
+With the Gravitino MCP server, users can manage and govern the metadata, as well as achieve metadata-driven actions by using natural language. Please follow issue [#7483](https://github.com/apache/gravitino/issues/7483) for more details. Also, you can check the [documentation](https://gravitino.apache.org/docs/1.0.0/gravitino-mcp-server) to know how to start the MCP server locally or in Docker.
+
+## Unified access control framework
+
+Gravitino introduced the RBAC system in the previous version, but it only offers users the ability to grant privileges to roles and users, without enforcing access control when manipulating the secure objects. In 1.0.0, we complete this missing piece in Gravitino.
+
+For now, users can set access control policies through our RBAC system and enforce the control when accessing secure objects. For the details, you can check out the umbrella issue [#6762](https://github.com/apache/gravitino/issues/6762).   
```

Comment:
Suggested change:
Currently, users can set access control policies through our RBAC system and enforce these controls when accessing secure objects. For details, you can refer to the umbrella issue [#6762](https://github.com/apache/gravitino/issues/6762).